internal/api/accounts.go

```diff
@@ -8,9 +8,17 @@ import (
 type AccountsClient interface {
 	Get(ctx context.Context) (*AccountResponse, error)
 	Update(ctx context.Context, data AccountUpdate) error
+	UpdateSettings(ctx context.Context, data AccountSettingsUpdate) error
 	Delete(ctx context.Context) error
 }
 
+// Settings is a representation of an account's settings.
+type Settings struct {
```

Comment:
i might call this `AccountSettings` to avoid any naming collisions in the future, bc all the structs in the `api` module share scope

Reply:
Great call, changed in c2aa1574c55ea687.